core/build.gradle.kts

```
@@ -146,10 +146,10 @@ java {
     targetCompatibility = JavaVersion.toVersion(jvmVersion)
 }
 
-if (isReleaseVersion) {
-    tasks.withType(Sign::class) {
-    }
+tasks.withType(Sign::class) {
+    onlyIf("isReleaseVersion is set", { isReleaseVersion } )
```

I think it may be idiomatic in Kotlin to put a lambda appearing as last parameter outside the parenthesis
```suggestion
    onlyIf("isReleaseVersion is set") { isReleaseVersion }
```